go.mod

```diff
@@ -107,7 +108,9 @@ require (
 	github.com/Masterminds/goutils v1.1.1 // indirect
 	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
+	github.com/Microsoft/go-winio v0.6.1 // indirect
```

Comment:
Very suspicious import, I cannot understand where it comes from :see_no_evil:

Reply:
((

```
go mod why github.com/Microsoft/go-winio
 
# github.com/Microsoft/go-winio
github.com/anyproto/anytype-heart/core/payments
github.com/wealdtech/go-ens/v3
github.com/ethereum/go-ethereum/accounts/abi/bind
github.com/ethereum/go-ethereum/accounts/external
github.com/ethereum/go-ethereum/rpc
github.com/Microsoft/go-winio
```